generators/client/templates/vue/package.json.ejs

```diff
@@ -117,6 +117,9 @@ limitations under the License.
 <%_ if (protractorTests) { _%>
     "protractor": "5.4.4",
 <%_ } _%>
+    <%_ if (cypressTests) { _%>
+      "cypress": "4.11.0",
```

Comment:
Latest?

Reply:
from when I checked, yes, but maybe it has been updated.

Reply:
oh yes 4.12.1 has been released ;)